src/Sarif/Converters/ClangAnalyzerConverter.cs

```diff
@@ -43,7 +43,6 @@ public void Convert(Stream input, IResultLogWriter output)
 
                 using (XmlReader xmlReader = XmlReader.Create(input, settings))
                 {
-                    XmlNodeType nodeType = xmlReader.MoveToContent();
```

Comment:
I'm concerned about this one. I understand that the return value from `MoveToContent` -- the variable `nodeType` -- is not used. But presumably the call to `MoveToContent` has side effects (changing the internal state of the reader). Do the tests pass when you remove this call?